Candidates.txt

```diff
@@ -1,3 +1,2 @@
 Bogdan Djuric bogdan-djuric
-Aleksandra Milenkovic Aleksandra-Milenkovic1
-Maja Dramicanin maja-dramicanin
+Ana Mucibabic Ana-Mucibabic
```

Comment:
That's fine, but you didn't merge this file as expected. You shouldn't remove some rows from that file, you should just add your at the end of this file (at next empty line)

Reply:
Can you please review it again? I did another commit